Assets/Scenes/Livingroom.unity

```diff
@@ -0,0 +1,8542 @@
+%YAML 1.1
```

Comment:
As per Grace's suggestion about the classname, this should likely also be renamed to "LivingRoom.unity"